tests/unit/certGen.unitTest.ts

```diff
@@ -8,14 +8,24 @@ import {S3BucketMockService} from "../models/S3BucketMockService";
 import {LambdaMockService} from "../models/LambdaMockService";
 import {CertificateUploadService} from "../../src/services/CertificateUploadService";
 import {ManagedUpload} from "aws-sdk/clients/s3";
+import {certGen} from "../../src/functions/certGen";
+import mockContext from "aws-lambda-mock-context";
+import sinon from "sinon";
+// tslint:disable
+const queueEventPass = require("../resources/queue-event-pass.json");
```

Comment:
possibly all requires may be replaced with readfile (before/beforeach as suitable)